src/client/components/DashboardCalendar.tsx

```diff
@@ -1,20 +1,122 @@
-import React from "react";
-import { Card, TextField, Button, CardContent } from "@material-ui/core";
+import React, { useState } from "react";
+import moment from "moment"
 
-interface DashboardCalendarProps{
+import { Card, CardContent } from "@material-ui/core";
+import { Calendar, momentLocalizer, View, DateLocalizer } from "react-big-calendar"
 
+import { EventsModal } from "./Modals"
+
+import * as dates from "../../../node_modules/react-big-calendar/lib/utils/dates"
+
+import "!style-loader!css-loader!react-big-calendar/lib/css/react-big-calendar.css"
+
+{/* 
+====================================
+            READ ME
+Modules required for implementation:
+====================================            
+
+react-big-calendar
+moment
+
+*/}
```

Comment:
We don't really need a READ ME, you can see the required modules in the import statements